README.md

```diff
@@ -1,6 +1,11 @@
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
```

Comment:
Are you sure you have not deleted Travis readme badge?

Reply:
@WillyBoris I don't think so since you PR hasn't been merged with develop and isn't even highlighted on the files changed in the READme. Or I'm I wrong?